packages/aws-rfdk/lib/deadline/scripts/python/test/test_configure_identity_registration_settings.py

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/env python3
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Tests for configure_identity_registration_settings.py
+"""
+
+import sys
+import unittest
+from unittest.mock import MagicMock, Mock
+
+sys.modules['boto3'] = Mock()
+
+# Python 3's modules don't make it easy to import from a directory above. This solves that.
+import pathlib
+_parentdir = pathlib.Path(__file__).parent.parent.resolve()
+sys.path.insert(0, str(_parentdir))
+import configure_identity_registration_settings as subject
+sys.path.remove(str(_parentdir))
```

Comment:
This can be simplified to:

```suggestion
import configure_identity_registration_settings as subject
```

but you need to make a few changes:

1.  Modify the `-s` to be up a directory in `packages/aws-rfdk/test-python.sh`:

    ```sh
    python3 -m unittest discover -s "$PWD/lib/deadline/scripts/python" -b
    ```
2.  Add an empty `packages/aws-rfdk/lib/deadline/scripts/python/test/__init__.py` file to mark the `test` directory as a python package

Reply:
Well that was some python wizardry I didn't know about. Thanks!